app/styles/typography/_font-weight-scale.scss

```diff
@@ -0,0 +1,16 @@
+// These will live in design system after we ensure everything looks good in mobile-wiki
+.wds-font-light {
+	font-weight: 300;
```

Comment:
we have $wds-fandom-font-weight-* variables in DS already that have the same values. Should we use them here?